pom.xml

```diff
@@ -1014,58 +986,64 @@
                                     <artifactId>mockito-errorprone</artifactId>
                                     <version>${version.mockito}</version>
                                 </path>
+                                <path>
+                                    <groupId>com.google.errorprone</groupId>
+                                    <artifactId>error_prone_core</artifactId>
+                                    <version>${version.error-prone}</version>
+                                </path>
                             </annotationProcessorPaths>
+                            <!-- Required to work with JDK 16+;
+                                see also https://errorprone.info/docs/installation#jdk-16. -->
+                            <fork>true</fork>
                             <compilerArgs combine.children="append">
-                                <!-- Enable and configure Error Prone. -->
-                                <!-- XXX: The awkward comment formatting used
-                                here ensures that the plugin arguments are
-                                separated only by spaces. Once we drop support
-                                for JDK 8 we can clean this up; later versions
-                                do properly handle newline separators. See
-                                https://github.com/google/error-prone/pull/1115.
-                                The SORTPOM processing instructions prevent the
-                                `sortpom-maven-plugin` from undoing the
-                                carefully applied formatting. -->
-                                <?SORTPOM IGNORE?>
-                                <arg>
-                                    -Xplugin:ErrorProne <!--
-                                    We use a separate Maven profile to raise
+                                <arg>-Xplugin:ErrorProne</arg>
+                                <!-- We use a separate Maven profile to raise
                                     all warnings as errors. This way one can
                                     optionally collect all warnings without
                                     failing the build on the first error
-                                    encountered.
-                                    --> -XepAllErrorsAsWarnings <!--
-                                    We want to enable almost all Error
+                                    encountered. -->
+                                <arg>-XepAllErrorsAsWarnings</arg>
```

Comment:
For some reason this causes `error: invalid flag: -XepAllErrorsAsWarnings` on Jolo. 😕

Reply:
In PSM we don't wrap every flag in `<arg>..</arg>` so I tried it there and got the same error message as you mention here.
I found a message posted by @Stephan202 [here](https://github.com/immutables/immutables/pull/1247#discussion_r529000593) with a small summary:

> All -Xplugin:ErrorProne arguments must be specified inside the same <arg>. So on JDK 8 one either has to put them all on one line (like here) or hide the newlines inside XML comments, like so.

So I'd say that reverting the `<arg>` changes should fix the issue 😄.

Reply:
Indeed. More generally: where possible we should match the setup of the Picnic-internal Maven parent.